examples/plot_metrics_with_error.py

```diff
@@ -0,0 +1,163 @@
+# %%
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+================================
+Plotting Metrics with Error Bars
+================================
+"""
+# %%
+# Load and preprocess the data set
+# --------------------------------
+# We download the data set using `fetch_adult` function in `fairlearn.datasets`.
+# We start by importing the various modules we're going to use:
+#
+
+import numpy as np
+import pandas as pd
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.metrics import recall_score
+from sklearn.datasets import fetch_openml
+from sklearn.metrics import confusion_matrix
+from fairlearn.metrics import MetricFrame, ErrorPlotter
+
+
+data = fetch_openml(data_id=1590, as_frame=True)
+X = pd.get_dummies(data.data)
+y_true = (data.target == '>50K') * 1
+sex = data.data['sex']
+
+# %%
+# Error Metrics
+# -------------
+# We have many different choices for error metrics. In this notebook we'll just be using a
+# Normal approximation interval (symmetric) and a Wilson score interval (asymmetric).
+#
+# https://en.wikipedia.org/wiki/Binomial_proportion_confidence_interval#Wilson_score_interval
+# https://en.wikipedia.org/wiki/Binomial_proportion_confidence_interval#Normal_approximation_interval
+#
+#
+
+z_score = 1.959964
```

Comment:
Where's that coming from?

Reply:
I would prefer to have a reference to this number in a comment. Also, there's a typo in the line above.